### ISSUE TITLE
Nested bitfields

### ISSUE THREAD

**Issue by HadrienRenaud** [HadrienRenaud/herdtools7#25]

**Comment by HadrienRenaud** [comment]
Fixed by herd/herdtools7#633.

**Comment by HadrienRenaud** [comment]
After checking, there is one more possibility not yet taken: a bitfield specified by a type. For example:
```
type myBits of bits(32) {
    [1:0] f,
    [1:0] nested: bits(2) {
        [1:0] g
    }
};
```

**Comment by HadrienRenaud** [comment]
Re fixed by herd/herdtools7#640
